.github/workflows/stale.yml

```diff
@@ -10,14 +10,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v1.1.0
+    - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-issue-message: 'This issue is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days'
         stale-pr-message: 'This pr is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days'
-        days-before-stale: 60
-        days-before-close: 15
+        days-before-stale: 90
+        days-before-close: 21
         stale-issue-label: 'lifecycle/stale'
         stale-pr-label: 'lifecycle/stale'
-        exempt-issue-label: 'lifecycle/frozen'
-        exempt-pr-label: 'lifecycle/frozen'
+        exempt-issue-label: 'lifecycle/frozen,type/enhancement,type/suggestion,type/bug'
+        exempt-pr-label: 'lifecycle/frozen,type/enhancement,type/bug-fix'
```

Comment:
After referring to docs, these fields should be updated:

- `exempt-issue-label` => `exempt-issue-labels`
- `exempt-pr-label` => `exempt-pr-labels`

Reply:
Fixed